README.md

```diff
@@ -32,14 +32,22 @@ load(
     "@io_bazel_rules_dotnet//dotnet:csharp.bzl",
     "csharp_repositories",
     "csharp_configure",
+    "nuget_package",
 )
 csharp_repositories()
 csharp_configure()
+
+nuget_package(
+  name="some_name",
+  pkg_name="Some.Package",
```

Comment:
Would it be better if we name the attributes `package` and `version` rather than `pkg_name` and `pkg_version`?